src/test/java/com/wikia/webdriver/testcases/adstests/TestSlotsOasis.java

```diff
@@ -0,0 +1,29 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEng
+ */
+public class TestSlotsOasis extends TemplateNoFirstLoad {
+
+  @Test(
+      groups = {"TestSlotsOasis"},
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "adsSlotsOasis"
+  )
+  public void adsSlotOasis(String slotName) {
```

Comment:
I would think about changing test method name to something more specific which will tell more about the test